knack/log.py

```diff
@@ -129,19 +129,23 @@ def configure(self, args):
         self.log_level = self._determine_log_level(args)
         console_log_levels = self._get_console_log_levels()
         console_log_formats = self._get_console_log_formats()
+
         root_logger = logging.getLogger()
-        cli_logger = logging.getLogger(self.cli_logger_name)
         # Set the levels of the loggers to lowest level.
         # Handlers can override by choosing a higher level.
         root_logger.setLevel(logging.DEBUG)
-        cli_logger.setLevel(logging.DEBUG)
-        cli_logger.propagate = False
-        if root_logger.handlers and cli_logger.handlers:
-            # loggers already configured
+
+        cli_loggers = [logging.getLogger(logger_name) for logger_name in cli_logger_names]
+        for cli_logger in cli_loggers:
+            cli_logger.setLevel(logging.DEBUG)
+            cli_logger.propagate = False
+
+        if root_logger.handlers:
+            # handlers already configured
             return
```

Comment:
Is it enough when `root_logger.handlers` is set? 
In previous code, both `root_logger.handlers` and `cli_logger.handlers` are checked. 
Should we check all `cli_logger.handlers for cli_logger in cli_loggers`?

Reply:
Note really necessary, as root logger and cli logger are always set at the same time. https://github.com/microsoft/knack/blob/30592e0b55e74dd1a57a73062155e44a744093c0/knack/log.py#L171-L177